modules/post/android/local/koffee.rb

```diff
@@ -0,0 +1,241 @@
+# frozen_string_literal: true
+
```

Comment:
This should be removed so the standard splat (the four lines below) are the top lines.

Reply:
Rubocop adds this automatically, as it's meant to increase performance long-term

https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/

Reply:
Cool thanks for clarifying @adfoster-r7. I'll mark this as resolved.